includes/pureclarity-template.php

```diff
@@ -55,20 +55,8 @@ public function __construct( &$plugin ) {
 	 * Renders configuration json
 	 */
 	public function render_pureclarity_json() {
-		$style = '';
-		if ( $this->is_pureclarity_active()
```

Comment:
do we not still have if PC is enabled?

Reply:
This if statement was only for outputting js to replace search results with our own results. It's no longer needed